packages/xml-views-validation/src/validators/elements/use-of-deprecated-class.ts

```diff
@@ -13,40 +14,52 @@ export function validateUseOfDeprecatedClass(
     return [];
   }
 
-  if (ui5Class.deprecatedInfo !== undefined) {
+  if (
+    ui5Class.deprecatedInfo !== undefined &&
+    // An issue without a position is not a useful issue...
+    (xmlElement.syntax.openName !== undefined ||
+      /* istanbul ignore next - defensive programing for (currently) none reproducible scenario */
+      //   We can't reach this branch because if openName does not exist, the FQN would not exist either
+      //   and thus the `ui5Class.deprecatedInfo` would not be found.
+      xmlElement.syntax.closeName !== undefined)
+  ) {
+    const deprecatedInfo = ui5Class.deprecatedInfo;
     const issues: UseOfDeprecatedClassIssue[] = [];
-    const commonIssueFields: UseOfDeprecatedClassIssue = {
-      kind: "UseOfDeprecatedClass",
-      message: `Deprecated UI5 Class: ${elementTagFqn} used.`,
-      severity: "warn",
-      // eslint-disable-next-line @typescript-eslint/no-explicit-any
-      range: undefined as any
+
+    const commonIssueFields = {
```

Comment:
I think you don't need this anymore since you only use it for`deprecatedIssue` which is already of the correct type (and has the ranges property)

Reply:
Yep thanks.